cinema/views.py

```diff
@@ -118,12 +148,18 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin
+):
     queryset = Order.objects.prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
```

Comment:
The `OrderViewSet` uses `IsAuthenticated` for `permission_classes`, which is different from the other viewsets that use `IsAdminOrIfAuthenticatedReadOnly`. Ensure this is intentional and aligns with the intended permission logic for orders.